packages/modular-scripts/src/build.ts

```diff
@@ -0,0 +1,488 @@
+// While working on this file, be aware that builds
```

Comment:
annoying that github is hiding this file because it's 'too big'.

Reply:
Could we break this up to make to it easier to grok / review in GitHub then?

Reply:
It's 500 lines of commented procedural code grouped by function, it's not that much. This is on github to fix.

Reply:
Well, maybe I could split it up. Ok, I promise to think strongly about it.